english/call-a-doctor/doctor.md

```diff
@@ -0,0 +1,12 @@
+You take a deep breath, your chest puffing out against your restrictive jacket.
+As you exhale, the walls stiffen, becoming less like marshmallow, and more like 
+the interior of your cell. The walls are a toasted-marshmallow kind of dirty white 
+color, not like the institutionalized white you remember from movies.
+
+The doctor enters.
+
+["Let me outta here, Doc!!!"](let-me-out.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.